packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -297,6 +299,35 @@ export class MetriportMedicalApi {
     });
   }
 
+  /**
+   * Returns the patients associated with given facility.
+   *
+   * @param facilityId The ID of the facility, optional. If not provided, patients from all facilities
+   *                   will be returned.
+   * @param filters Full text search filters, optional. If not provided, all patients will be returned
+   *                (according to pagination settings).
+   *                See https://docs.metriport.com/medical-api/more-info/filters
+   * @param pagination Pagination settings, optional. If not provided, the first page will be returned.
+   *                   See https://docs.metriport.com/medical-api/more-info/pagination
+   * @return The list of patients.
+   */
+  async listNetworks({
```

Comment:
id call this listNetworkItems or listNetworkEntries something along those lines

Reply:
yep thats clearer for sure

Reply:
addressed